runtime/runtime/src/actions.rs

```diff
@@ -458,7 +457,7 @@ pub(crate) fn action_delete_account(
             // account storage usage should be larger than code size
             let code_len = code.code.len() as u64;
             debug_assert!(account_storage_usage > code_len);
```

Comment:
We don't run `debug_assert` in prod, right? So it's only for tests

Reply:
yes